source/agora/node/Ledger.d

```diff
@@ -827,6 +865,8 @@ version (unittest)
     /// A `Ledger` with sensible defaults for `unittest` blocks
     private final class TestLedger : Ledger
     {
+        public MockClock mock_clock;
+
```

Comment:
Why redefine it here ? Just use `Ledger.clock` ?

Reply:
Ledger.clock is private unfortunately and I need to access it from outside Ledger.d for testing. 

I was thinking to make it public only in debug modes. Should I do that instead of keeping a reference to the clock in the TestLedger?

Reply:
ok, I will just make it a constructor parameter instead, it is more clear